batch/oak_ffc_4p_poe.json

```diff
@@ -5,6 +5,11 @@
     "test_type": "OAK-1-POE",
```

Comment:
Test type OAK-1-PoE is correct?

Reply:
Yes. This parameter isn't used anymore and will be removed in the future.